.github/workflows/maven-build.yml

```diff
@@ -104,6 +104,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: mvn clean package -am
+        with:
+          max_attempts: ${{ github.ref == 'refs/heads/main' && 3 || 1 }}
```

Comment:
This will not work! Please refer to the [github actions workflow syntax](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions)!

Github does not provide out of box way to rerun a step :(  You might want to look for [github actions from marketplace](https://github.com/marketplace?type=actions)!

Reply:
fixed

Reply:
Thanks @sushi30